src/lib/SpotlightTour.provider.tsx

```diff
@@ -117,7 +123,7 @@ export const SpotlightTourProvider = React.forwardRef<SpotlightTour, SpotlightTo
         ? stop()
         : renderStep(current + 1);
     }
-  }, [renderStep, current, steps.length]);
+  }, [stop, renderStep, current, steps.length]);
```

Comment:
Just checking if `step` should also be added as dep?

Reply:
No, `step` is only in `renderStep` function scope 🙂